eZ/Publish/Core/Limitation/ObjectStateLimitationType.php

```diff
@@ -137,8 +137,8 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
         $objectStateHandler = $this->persistence->objectStateHandler();
         $stateGroups = $objectStateHandler->loadAllGroups();
 
-        // First deal with unpublished content
-        if ($object instanceof ContentCreateStruct || !$object->published) {
+        // First deal with new content
+        if ($object instanceof ContentCreateStruct) {
```

Comment:
Should we also add a comment here that ATM there is no known use-case where `$object` might ever be a `ContentCreateStruct` ? ( There is no object state limitation for `content/create`, only for `content/edit`